components/event-simulator/org.wso2.carbon.event.simulator.core/src/main/java/org/wso2/carbon/event/simulator/core/internal/CarbonEventSimulator.java

```diff
@@ -56,6 +56,7 @@
 import java.sql.SQLException;
 import java.sql.Statement;
 import java.util.*;
```

Comment:
Better to change this "java.util.*" to specific imports, while the imports are being changed.